vendor/jquery.jfeed.js

```diff
@@ -134,6 +134,9 @@ JFeed.prototype = {
 
             this.type = 'atom';
             var feedClass = new JAtom(xml);
+        } else {
+
```

Comment:
empty line

Reply:
It was deliberate, just like the tab is 3 spaces, to mimic the rest of this library file.